src/m365/aad/commands/app/app-add.ts

```diff
@@ -68,11 +70,18 @@ interface Options extends GlobalOptions {
   certificateDisplayName?: string;
 }
 
+interface AppPermissions {
+  ResourceId: string;
+  resourceAccess: ResourceAccess[];
+  Scope: string[];
```

Comment:
I think that in order to be consistent we should properties to lowerCamelCase. Could you align the `resourceId` and `scope`